healthcare/2.22/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,36 @@
+* https://www.anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%202%20-%20CMS%20Interoperability/[CMS Interoperability^] — Includes assets to help you meet CMS Interoperability and Patient Access final rule.
+* https://www.anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%203%20-%20Labs%20integration/[Labs integration^] — Includes the The Electronic Laboratory Reporting (ELR) to CalREDIE implementation template, which helps you transmit laboratory reports that identify a notifiable condition to public health agencies (for example, California Department of Public Health).
+* https://www.anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%204%20-%20Appointment%20scheduling/[Appointment scheduling^] — Includes the Appointment Scheduling Process API for both Cerner and Epic, which enables call center agents to schedule, update, and cancel appointments on behalf of the patient. See the implementation template for more information.
+* https://www.anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%205%20-%20Benefits%20and%20eligibility%20verification/[Benefits and eligibility verification^] — Enables call center agents and care staff to perform real-time benefits and eligibility verification. Providers can electronically verify a patient's insurance coverage in real-time for medical treatment directly within Salesforce Health Cloud.
+* https://www.anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%206%20-%20Prior%20Authorization/[Prior Authorization^] — Includes CRD, DTR, and PAS API specifications and implementation templates to automate the prior authorization process for more immediate authorizations, leading to better outcomes and increased satisfaction for patients, providers, and payers.
+* https://www.anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%207%20-%20Population%20health%20management/[Population health management^] -  Convert and surface valuable population health data into Amazon HealthLake to improve clinical outcomes.
+* https://www.anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%207%20-%20Qualified%20health%20information%20network%20integration/[Qualified Health Information Network Integration] - Streamline patient onboarding and remove manual data entry by integrating with external organizations to gather and ingest patient record data.
```

Comment:
Can we make this line read: 
"Streamline patient onboarding and remove manual data entry by integrating with external healthcare organizations to gather and ingest patient data."